File: Python PowerPoint Examples/PropertiesApiDemo/setPptDocumentProperty.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.properties_api.PropertiesApi(configuration)

name = "SetPptDocumentProperty.pptx"  # Supported formats:ppt/pptx/pptm/pps/ppsx/pot/potx/odp
propertyName = "Keywords"
password = ""
storage = ""
folder = "input"
_property = spirecloudpowerpoint.models.DocumentProperty("Keywords", "SetPptDocumentProperty_pptx", False)

api.set_ppt_document_property(name, propertyName, _property=_property, password=password,
                              storage=storage, folder=folder)

File: JavaScript PowerPoint Examples/DocumentPropertiesApiDemo/setPptDocumentProperty.js
(function (){
    var appId = "your ID";
    var appKey = "your Key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudPowerpoint = require('../../src/index');
    var apiClient = new SpirecloudPowerpoint.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudPowerpoint.PropertiesApi(apiClient);

    var name = "SetPptDocumentProperty.pptx";
    var propertyName = "Keywords";
    var property = new SpirecloudPowerpoint.DocumentProperty();
    property.Name = "Keywords";
    property.Value = "SetPptDocumentProperty_pptx";
    property.BuiltIn = false;
    var opt = { 'property': property, 'folder': "input" };
    var response = instance.setPptDocumentProperty(name, propertyName, opt,
        function (error, data, response) {
            if (error) throw error;
        });
})();